pymongo/asynchronous/mongo_client.py

```diff
@@ -1709,6 +1710,11 @@ async def _get_topology(self) -> Topology:
         If this client was created with "connect=False", calling _get_topology
         launches the connection process in the background.
         """
+        if not _IS_SYNC:
+            if self._loop is None:
+                self._loop = asyncio.get_running_loop()
+            elif self._loop != asyncio.get_running_loop():
+                raise RuntimeError("Cannot use AsyncMongoClient in different event loop")
```

Comment:
Are there any docs we can point to in this error message? Or can we make it a bit more explanatory?